docs/source/examples/beam_output.txt

```diff
@@ -0,0 +1,26 @@
+
+Cost
+----
+ 0.7812 [m] 
+
+Free Variables
+--------------
+ M : [ 1.25e+03  988       756       556      ... ]  [N*m] Internal moment
+ V : [ 500       444       389       333      ... ]  [N]   Internal shear 
+th : [  -        0.0622    0.111     0.147    ... ]        Slope          
+ w : [  -        0.0173    0.0653    0.137    ... ]  [m]   Displacement   
+
+Constants
+---------
+EI : 1e+04                                           [N*m**2]                
+dx : 0.5556                                          [m]                     
+ M : [  -         -         -         -       ... ]  [N*m]    Internal moment
+ V : [  -         -         -         -       ... ]  [N]      Internal shear 
+th : [ 1e-16      -         -         -       ... ]           Slope          
+ w : [ 1e-16      -         -         -       ... ]  [m]      Displacement   
```

Comment:
hmm, substituting one element of a vector variable sure makes for overly-complicated output. Not your fault though!